app/models/text_annotation.rb

```diff
@@ -2,6 +2,8 @@ class TextAnnotation < Annotation
 
   validates_presence_of :line_start
   validates_presence_of :line_end
+  validates_presence_of :column_start
+  validates_presence_of :column_end
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.